src/posix.mak

```diff
@@ -204,6 +220,12 @@ SRC = win32.mak posix.mak osmodel.mak \
 	ctfe.h ctfeexpr.c visitor.h nspace.h nspace.c errors.h errors.c \
 	escape.c tokens.h tokens.c globals.h globals.c
 
+ifeq ($(D_OBJC),1)
```

Comment:
This should be unconditional as it's only used for generating zips, not building.